Source/World/World.cpp

```diff
@@ -0,0 +1,118 @@
+//
+// Am I god if I have created the World Class? - god
+//
+
+#include "World.h"
+
+//If no user configuration is set call the default constructor
+
+World::World()
+{
+	srand( (unsigned)time( NULL ) );
```

Comment:
@theawesomechin  I made an interface for random numbers, which we could use instead. I will make this more flexible because it will help us test the random numbers.